taf/OT/OT.py

```diff
@@ -44,7 +44,7 @@ def AWS_Extract_Line(self, TMSIS_SCHEMA, DA_SCHEMA, fl2, fl, tab_no, _2x_segment
         # Create a temporary line file
         z = f"""
             create or replace temporary view {fl2}_LINE_IN as
-            select
+            cache select
```

Comment:
Was this change intentional?

Reply:
Nope. Was working on something locally and that snuck in. Thanks.